src/locales/se.js

```diff
@@ -19,19 +19,22 @@ const se = {
       size: 'Hur stort är ditt team?',
       officeType: 'Vilken typ av kontor letar du efter?',
       name: 'Vad är namnet på erat företag?',
-      peers: 'Vill du dela kontor med andra från din bransch?'
+      peers: 'Vill du dela kontor med andra från din bransch?',
+      location: 'Vilka områden är du intresserad av?',
     },
 
     answer: {
-      email: 'Min email är: ',
-      size: 'Vi är en grupp av: ',
-      officeType: 'Vi behöver: ',
+      email: 'Min email är... ',
+      size: 'Vi är en grupp av... ',
+      officeType: 'Vi behöver... ',
       officeLable: 'Kontor',
       openSpaceLable: 'Öppet utrymme',
-      name: 'Det heter: ',
-      companyPlaceholder: 'Mitt företag',
+      name: 'Det är... ',
+      companyPlaceholder: 'MyCoolCompany',
```

Comment:
Not Swedish but i think its fine but can you try to come up with a Swedish phrase here?

Reply:
Can take a look at it.

Reply:
Changed